flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-values/src/main/java/org/apache/flink/cdc/connectors/values/sink/ValuesDataSinkOptions.java

```diff
@@ -35,4 +35,10 @@ public class ValuesDataSinkOptions {
                     .booleanType()
                     .defaultValue(true)
                     .withDescription("True if the Event should be print to console.");
+
+    public static final ConfigOption<Boolean> LEGACY_ENABLED =
+            ConfigOptions.key("legacy.enabled")
```

Comment:
The word `legacy` is too vague. Sink V1 could also be a legacy one. What about using an enumeration here listing all APIs that the value sink supports?

Reply:
Add `SinkType`